distribution/ddf-common/src/main/resources/bin/ddfsolr

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+# Capture start|stop command
+# Set a default value in the event no arguments are provided
+COMMAND=""
+COMMAND=$1
+shift
+
+# In the secure configuration, Solr will only accept two-way TLS
+# connections. Additionally, Solr will reject any remote requests;
+# requests that do not originate from the local host.
+SECURE=true
+
+# -----------------------------------------------------------------------------
+usage() {
+    # -u Unsecure. Do not use TLS. Acceptremote requests
+    echo "Usage: $0 {start|stop} [-u]" >&2; exit 1;
+}
+
+while getopts ":u" o; do
+    case "${o}" in
+        u )
+            SECURE=false
+            ;;
+
+        * )
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+# -----------------------------------------------------------------------------
+if [ ! $COMMAND ]; then
```

Comment:
You might want to change this to `if [ -z "$COMMAND" ]; then` which will return true if the variable is either unset or set to an empty string